contracts/implementation/Resolver.sol

```diff
@@ -0,0 +1,24 @@
+//SPDX-License-Identifier: CC-BY-NC-ND-4.0
+pragma solidity 0.8.7;
+
+interface IPoolKeeper {
```

Comment:
According to this: https://docs.gelato.network/guides/writing-a-resolver/smart-contract-resolver
it should implement `IResolver` (probably just an interface with `checker` function would be my guess).